src/installer/corehost/cli/fxr/hostfxr.cpp

```diff
@@ -636,7 +636,11 @@ namespace
 // Return value:
 //     The error code result.
 //
-// The host_context_handle must have been initialized using hostfxr_initialize_for_runtime_config.
+// If the host_context_handle was initialized using hostfxr_initialize_for_runtime_config,
+// then all delegate types are supported.
+// If the host_context_handle was initialized using hostfxr_initialize_for_dotnet_command_line,
+// then only the following delegate types are currently supported:
+//     hdt_load_assembly_and_get_function_pointer
```

Comment:
This is something for the header file for documentation. I wouldn't add public policy decisions in the implementation since we inevitable need to rummage around and update both official documentation and implementation which is confusing. This support isn't hidden so let the code and official doc do the talking.

Reply:
I agree that this belongs in the header. However, all of this currently lives here. Should I delete all of the comments on public methods in this file and move them to hostfxr.h?

Reply:
Oh boo. I didn't realize all comments for APIs were here. You are free to leave this if following the existing convention. Thanks for the attention to detail.